.golangci.yml

```diff
@@ -0,0 +1,20 @@
+run:
+  timeout: 5m
+
+linters:
+  enable:
+    # Default linters
+    - errcheck     # Detect unchecked errors
```

Comment:
Replaced old linters with given, except two of them: 
- gofumpt. Reason: asserts "File is not `gofumpt`-ed" even for formatted files;
- noctx. Reason: cannot pass context into a scheduler properly.

Reply:
Is it OK?

Reply:
gofumpt. Reason: asserts "File is not gofumpt-ed" even for formatted files;  - ok. 

* noctx. Reason: cannot pass context into a scheduler properly. - for this task, I think, it will be ok.

You can read about `graceful shutdown` in your app. In this case, you can pass the context to all workers.